src/components/components/CountDown.js

```diff
@@ -39,6 +39,40 @@ export const getCountDisplay = (
   )}`;
 };
 
+export const getCountDisplayToHours = (
```

Comment:
Please remove the old function : `getCountDisplay` . 

It isn't used anywhere else.

Reply:
Will do.